lib/Sema/SemaDecl.cpp

```diff
@@ -8521,16 +8526,20 @@ Sema::ActOnFunctionDeclarator(Scope *S, Declarator &D, DeclContext *DC,
     }
   }
 
-  // Checked C - checked function return type checking
+  // Checked C - type restriction on declarations in checked block
```

Comment:
Please add periods 

// Checked C - type restriction on declarations in checked block.
// Function parameter & return are not allowed to use unchecked type
// in checked blocks.